source/content/drush-import.md

```diff
@@ -1,37 +1,46 @@
 
+Follow the steps below carefully to ensure that you import your Drupal site correctly.
+
 ## Before You Begin
 
-There are a few things you'll need in order to make this work:
+1. Create a Pantheon account with at least one free Dev site slot open. 
+
+   - A [Pantheon account is free](https://dashboard.pantheon.io/register), and if you need an extra Dev site to try this out, just ask and we'll be happy to grant you one.
 
-1. A Pantheon account with at least one free Dev site slot open. A [Pantheon account is free](https://dashboard.pantheon.io/register), and if you need an extra Dev site to try this out, just ask and we'll be happy to grant you one.
-2. A working local Drush installation that is up to date with 7.x or 8.x stable.
-3. Drush access to your existing Drupal site.
+1. Verify that you have Drupal 7 with Drush 8 if you have a non-Composer managed site *or* Drupal 9 with [Drush 11.1.1 or higher](https://www.drush.org/latest/install/) if you have a Composer-managed site.
+
+1. Verify that you have Drush access to your existing Drupal site.
 
 ## Generate a Drush Archive
 
-The first thing you'll need to do is to generate a Drush archive of your existing site. If you have Drush access to the site direct via the shell, it's easy with the archive-dump command:
+The first thing you'll need to do is to generate a Drush archive of your existing site. 
 
-```bash{promptUser: user}
-drush archive-dump --destination=drush-archive.tar.gz
-```
+1. Navigate to site root.
 
-Executed from the site root will create a file called drush-archive.tar.gz that's available via the public internet. If you have the file locally, you can put it on Dropbox, S3, or any number of other places. The important thing is that you have a Drush archive that can be downloaded via a URL.
+1. Run the command below if you have Drush access to the site direct via the shell:
 
-## Set Up Terminus
+   ```bash{promptUser: user}
+   drush archive-dump --destination=drush-archive.tar.gz
+   ```
 
-If you haven't already, set up Terminus, the Pantheon CLI, using Composer as described in the [installation instructions on GitHub](https://github.com/pantheon-systems/cli/wiki/installation).
+   - This creates a file called `drush-archive.tar.gz` that's available via the public internet. If you have the file locally, you can put it on Dropbox, S3, or any number of other places. The important thing is that you have a Drush archive that can be downloaded via a URL.
 
 ## Import Your Archive
 
-Importing a Drush site archive as we've prepared it above is easy.
+<TabList>
+
+<Tab title="Drupal 7 non-Composer" id="d7" active={true}>
+
+1. Set up Terminus, the Pantheon CLI, if you haven't already using Composer as described in the [installation instructions on GitHub](https://github.com/pantheon-systems/cli/wiki/installation).
```

Comment:
The Terminus 3 install instructions in the other tab should be fine for this use-case as well. Maybe we should pull these up, outside the tab group.